kythe/proto/xref.proto

```diff
@@ -265,6 +265,9 @@ message DecorationsReply {
   // references to the list of their overrides.
   map<string, Overrides> extends_overrides = 17;
 
+  // A unique build_id for the underlying dataset serving this reply.
```

Comment:
s/build_id/identifier/ in the doc comments here and elsewhere.

Reply:
done